src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -23,6 +24,7 @@
 import java.io.Reader;
 import java.lang.*;
 
+import java.util.*;
```

Comment:
Can we remove this, and replace with specific  imports for specific items?

Reply:
VSCode is reporting some of the other imports as "unused"... should those be stripped out too?